test/Serilog.Tests/Context/LogContextTests.cs

```diff
@@ -143,7 +144,7 @@ public void MultipleNestedPropertiesOverrideLessNestedOnes()
         // since user property types may not be serializable.
         // Fails if the Serilog assemblies cannot be loaded in the
         // remote domain. See also LogContext.Suspend()
-        [Test]
+        [Fact(Skip = "Fails on DNX451")]
```

Comment:
By "DNX451" do you mean "DNXCORE50"? I was under the impression 451 and .NET 4.5 were roughly equivalent. I am probably a bit behind :-)